owncloudApp/src/main/java/com/owncloud/android/ui/activity/Preferences.java

```diff
@@ -873,8 +875,7 @@ private AppCompatDelegate getDelegate() {
      * Load picture upload path set on preferences
      */
     private void loadCameraUploadsPicturePath() {
-        mUploadPath = mAppPrefs.getString(PREFERENCE_CAMERA_PICTURE_UPLOADS_PATH,
-                getString(R.string.camera_upload_path));
+        mUploadPath = mAppPrefs.getString(PREFERENCE_CAMERA_PICTURE_UPLOADS_PATH, PREF__CAMERA_UPLOADS_DEFAULT_PATH);
```

Comment:
This path is not the same as before, we are adding a path separator that before we didn't have. It is intended?

Reply:
I defined `PREF__CAMERA_UPLOADS_DEFAULT_PATH` as a constant in `PreferenceManager` and all the constants in that file already had that path separator you mean, so I followed the same format.

Reply:
I know, but previously camera uploads default path was '/CameraUpload' and now it is '/CameraUpload/'. And there were different uses, some with last file separator and some without. Now, we are using always with last path separator, So I meant that if that change was on purpose or not

Reply:
I see what you mean and just pushed a new commit with some changes, can you recheck it? Thanks